plugins/github-release-manager/src/cards/info/Info.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Link, Typography } from '@material-ui/core';
+
+import { Differ } from '../../components/Differ';
+import { InfoCardPlus } from '../../components/InfoCardPlus';
+import {
+  GhGetBranchResponse,
+  GhGetReleaseResponse,
+  Project,
+} from '../../types/types';
+import { useStyles } from '../../styles/styles';
+import { TEST_IDS } from '../../test-helpers/test-ids';
+import flowImage from './flow.png';
+
+interface InfoCardProps {
+  releaseBranch: GhGetBranchResponse | null;
+  latestRelease: GhGetReleaseResponse | null;
+  project: Project;
+}
+
+export const Info = ({
+  releaseBranch,
+  latestRelease,
+  project,
+}: InfoCardProps) => {
+  const classes = useStyles();
+
+  return (
+    <InfoCardPlus>
+      <div style={{ marginBottom: '1em' }} data-testid={TEST_IDS.info.info}>
+        <Typography variant="h6">Terminology</Typography>
+
+        <Typography>
+          <strong>GitHub</strong>: The source control system where releases
+          reside in a practical sense. Read more about GitHub releases{' '}
+          <Link
+            href="https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository"
+            target="_blank"
+          >
+            here
+          </Link>
```

Comment:
In general terms, web sites should always avoid linking to "here" both for user understanding (where our eyes scan for underlined words) and for accessibility reasons if a site reader is reading the words.  I'd suggest linking to the phrase "GitHub Releases" if possible and just avoid "here"?

Reply:
👍 

Addressed in https://github.com/backstage/backstage/pull/5299/commits/c003e06cc004cc085153112af6240a30e7d73200